app/views/pages/home.html.erb

```diff
@@ -33,23 +33,73 @@
       <h4 class="startpage-subtitle box__bg bg--white"><%= t(:quote_text, scope: 'pages.home') %></h4>
     </div>
   </section>
+  <!-- Anker für Kategorien -->
+  <a id="categories_anchor"></a>
   <div class="copyright">
     Copyright by Ute Mayer
   </div>
 
-  <!-- Anker für Kategorien -->
-  <a id="categories_anchor"></a>
-
   <div class="container">
-    <h1 class="box__bg"><%= t(:description, scope: 'pages.home') %></h1>
-    <h4 class="border-vertical d-block box__bg mb-4">
-      <%= t(:description_text, scope: 'pages.home') %>
-    </h4>
     <div class="row">
-      <%= render partial: "shared/categories", object: @categories %>
+      <div class="col-3">
+        <h5>Choose a category:</h5>
+        <div class="nav flex-column nav-pills" id="v-pills-tab" role="tablist" aria-orientation="vertical">
+          <% @categories.each_with_index do |category, index| %>
+            <a class="nav-link <%= "active show" if index == 0%>" id="v-pills-<%= category.short_name %>-tab" data-toggle="pill" href="#v-pills-<%= category.short_name %>" role="tab" aria-controls="v-pills-<%= category.short_name %>" aria-selected="false">
+              <i class="fa fa-circle pr-2 small"></i><%= category.name %>
+            </a>
+          <% end %>
+        </div>
+      </div>
+
+      <div class="col-9">
+        <h5>Choose one or more tags:</h5>
+        <div class="tab-content mb-3" id="available-tags-box">
+          <% @categories.each_with_index do |category, index| %>
+            <div class="tab-pane fade <%= "active show" if index == 0%>" id="v-pills-<%= category.short_name %>" role="tabpanel" aria-labelledby="v-pills-<%= category.short_name %>-tab" >
+              <%= render partial: "shared/tag_box", locals: { :tags => instance_variable_get("@tags_#{category.short_name}"), :category => category } %>
+            </div>
+          <% end %>
+        </div>
+        <div id="selected-tags-box" class="border my-3">
+        </div>
+        <div class="form-group">
+          <%= form_tag(profiles_path, method: "get") do %>
+            <%= text_field_tag :tag_search, params[:tag_search], id: "hidden_tag_input", class: "form-control d-none"   %>
+            <%= submit_tag "Search in topics", class: "btn btn-primary", id: "search_in_topic" %>
+            <span class="btn btn-primary" id="clear"><i class="fa fa-times pr-1"></i>Clear</span>
+          <% end %>
+        </div>
+      </div>
     </div>
   </div>
-
+  <script>
+    $( "#available-tags-box .available-tag" ).click(function() {
+      console.log("test");
+      $(this).clone().appendTo( "#selected-tags-box" ).append("<i class='pl-2 fa fa-times-circle'></i>").removeClass("available-tag").addClass("selected-tag");
+      $(this).addClass( "disabled" );
```

Comment:
```.unbind("click");```

Reply:
:+1:

Reply:
At the end I used this:
```
.no-click {
    pointer-events: none;
}
```

Because `unbind` is deprecated in jquery3 and the `bind` method did not work. Adding the css class was super simple and works. :tada: